hostports/watcher.go

```diff
@@ -109,6 +111,25 @@ func (p PortRule) natIptables() []byte {
 	return buf.Bytes()
 }
 
+func (p ChainRule) filterIptables() []byte {
```

Comment:
aren't these rules applicable in all the scenarios? do you have to figure out the routers and only then apply?